events/discordEvents/client/ready.js

```diff
@@ -834,6 +834,52 @@ module.exports = {
 
             sendToChannelFeeds('postUpdate', data, client);
          });
+
+         // =============================================================
+         // Lil Nouns
+         // =============================================================
+         
+         Nouns.on('ProposalCreated', async data => {
+            data.description = data.description.substring(0, 500);
+
+            try {
+               const proposal = await Proposal.tryCreateProposal(data);
+               data.proposalTitle = proposal.fullTitle;
+               if (data.proposalTitle.length >= MAX_PROPOSAL_TITLE) {
+                  data.proposalTitle =
+                     data.proposalTitle
+                        .substring(0, MAX_PROPOSAL_TITLE)
+                        .trim() + '...';
+               }
+            } catch (error) {
+               Logger.error('events/ready.js: Error creating a proposal.', {
+                  error: error,
+               });
+            }
+
+            Logger.info(
+               'events/ready.js: On ProposalCreated.',
+               {
+                  id: `${data.id}`,
+                  proposer: `${data.proposer.id}`,
+                  startBlock: data.startBlock,
+                  endBlock: data.endBlock,
+                  quorumVotes: `${data.quorumVotes}`,
+                  proposalThreshold: `${data.proposalThreshold}`,
+                  description: data.description,
+                  targets: `${data.targets}`,
+                  values: `${data.values}`,
+                  signatures: `${data.signatures}`,
+                  calldatas: `${data.calldatas}`,
+               },
+            );
+
+            data.nounsForumType = 'PropCreated';
+
+            // sendToChannelFeeds('newProposalPoll', data, client);
+            sendToChannelFeeds('propCreated', data, client);
```

Comment:
This will send the information to the Nouns DAO feed. We could either create a specific LilNouns feed for this event, or remove it to exclusively focus on the forums.